next/src/components/global/Footer/Footer.tsx

```diff
@@ -0,0 +1,61 @@
+import getLegalLinks from '@/utils/get-legal-links';
+import sanityFetch from '@/utils/sanity.fetch';
+import Logo from '@/components/ui/Logo';
+import SocialLink from '@/components/ui/SocialLink';
+import TextLink from '@/components/ui/TextLink';
+import styles from './Footer.module.scss';
+
+type FooterSocialsTypes = { _type: PlatformType; url: string }[];
+
+async function fetchSocials(): Promise<FooterSocialsTypes> {
+  return await sanityFetch({
+    query: /* groq */ `
+     *[_id == 'global'][0].footer.socialMedia
+    `,
+    tags: ['global'],
+  });
+}
+
+export default async function Footer() {
+  const { termsAndConditions, privacyPolicy } = await getLegalLinks();
+  const socials = await fetchSocials();
+
+  return (
+    <footer className={`${styles.footer} max-width`}>
+      <div className={styles.row}>
+        <Logo />
+        <nav className={styles.socials}>
+          {socials.map(({ _type, url }, i) => (
+            <SocialLink
+              style={{ zIndex: `${20 - i}` }}
```

Comment:
I completely don't know why you use there that strange `zIndex`?

Reply:
So that the arrow icon does not go behind the second SocialLink etc. 

![1](https://github.com/user-attachments/assets/491b1eed-5abd-4899-b42d-8395f03354cc)